argm.c

```diff
@@ -311,17 +311,25 @@ argm_combine_universal(PG_FUNCTION_ARGS, int32 compareFunctionResultToAdvance)
 
     GET_AGG_CONTEXT(fcinfo, aggcontext);
 
-    if (PG_ARGISNULL(1))
-        PG_RETURN_POINTER(PG_GETARG_POINTER(0));
+    state0 = PG_ARGISNULL(0) ? NULL : (ArgmState *) PG_GETARG_POINTER(0);
+    state1 = PG_ARGISNULL(1) ? NULL : (ArgmState *) PG_GETARG_POINTER(1);
```

Comment:
This is what we changed the bug point B into: even if we get a zero pointer (which we shouldn't now), treat it the same as an SQL null and don't attempt to use it.